web/src/render/useNotificationsRenderer.ts

```diff
@@ -91,6 +92,16 @@ export const useNotificationsRenderer: UsePixiRenderer = ({
   const { initialized } = useConnectionState();
   const debouncedBounds = useDebounce(bounds, 50);
   const swrKey = useNotificationsDataKey();
+  const [logNewNotifications, setLogNewNotifications] = useState<
+    boolean | null
+  >();
+
+  useEffect(() => {
+    if (logNewNotifications) {
+      trackAnalyticsEvent("new-notifications");
+      setLogNewNotifications(false);
```

Comment:
since you set it back to false here, then you will just continue to log all notifications :)

instead of doing this use a ref

```
const trackedNotifications = useRef(false);

...onSuccess: (newNotifications) => {
  if (newNotifications.length > 0) {
    if (!trackedNotifications.current) {
      trackAnalyticsEvent("new-notifications");
      trackedNotifications.current = true;
    }
    ...
```

Reply:
ooops, I forgot to remove it. That's better, thanks!